nomad/grapl-core.nomad

```diff
@@ -9,20 +9,29 @@ variable "container_registry" {
   description = "The container registry in which we can find Grapl services. Requires a trailing /"
 }
 
+variable "container_repo" {
+  type        = string
+  default     = ""
+  description = "The container repo inside the registry in which we can find Grapl services. Requires a trailing /"
+}
+
 variable "aws_access_key_id" {
   type        = string
+  default     = "DEFAULT_NOMAD_AWS_ACCESS_KEY_ID"
```

Comment:
Instead of using an empty string or "test" I'm setting an easily searchable string. Let me know if you've got better ideas here.